src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -336,34 +274,46 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
     navigateToValueBreakdown(ValueSlugs.field, value, serviceScene);
   };
 
-  public static Component = ({ model }: SceneComponentProps<FieldsBreakdownScene>) => {
-    const { body, loading, blockingMessage, search, sort } = model.useState();
+  public static ParentMenu = ({ model }: SceneComponentProps<FieldsBreakdownScene>) => {
```

Comment:
What is a parent menu?

Reply:
A poorly named menu :P

Reply:
See https://github.com/grafana/explore-logs/pull/936/commits/8feec08b0b26cb0773e299722172e14fe1ace406